config.go

```diff
@@ -160,6 +161,7 @@ type config struct {
 	PipeRx               uint          `long:"piperx" description:"File descriptor of read end pipe to enable parent -> child process communication"`
 	PipeTx               uint          `long:"pipetx" description:"File descriptor of write end pipe to enable parent <- child process communication"`
 	LifetimeEvents       bool          `long:"lifetimeevents" description:"Send lifetime notifications over the TX pipe"`
+	PeerIdleTimeout      time.Duration `long:"peeridletimeout" description:"Duration of inactivity before a peer is timed out"`
```

Comment:
description needs unit of time specified.

Reply:
Done. But what about the minimum time? see comment below.